### DISCUSSION TITLE
a problem with starting webui

### DISCUSSION THREAD

**Discussion by NaotoEnjoyer** [TheLastBen/fast-stable-diffusion#1089] [Q&A] [answered]
hey ill keep it short, i did everything my friend showed me, maunted, installed webui on my gdrive, the reqs next, then put models path and tick model load section and after all that started last section, start webui, and after it installed start on first launch it gave two links, local and public gradio one, local one doesnt open webui, gives error like when i didnt launch my local pc webui and tried to open loacl ip, public gradio link just gives me white blank page and thats all
![image](https://user-images.githubusercontent.com/117005058/208536334-1e213902-d602-4670-93bd-2699499b5dfb.png)
![image](https://user-images.githubusercontent.com/117005058/208536367-7d9f9470-b7fe-4587-97a3-dd2f717343a0.png)

**Comment by Excalibro1** [comment, marked as answer]
Untick the "use gradio server" and run again it will use local tunnel which should work.

**Reply by NaotoEnjoyer** [reply]
ty now it works, but after i got all stuff in place and started to generate images webui after gen doesnt give image results in preview windos its empty, i have to re open webui link each time i did generation otherwise after gen it doesnt show result but save it on gdrive and doesnt let me generate images again until i close tab and reopen again and this ofc resets webui state, whats wrong? heres how it looks, after generation result window is stuck like this and this softlocks generate button until i refresh webui page, and webui does save images on gdrive it works, but webui glitches like this for some reason
![image](https://user-images.githubusercontent.com/117005058/208629963-6f5ff197-0f39-4d48-81be-58a015ef7454.png)

**Reply by Excalibro1** [reply]
Sadly I think that's a problem with the local tunnel I believe the gradio server doesn't encounter these issues, maybe try the gradio server now It might work lol

A thread where it's talked about:
https://github.com/TheLastBen/fast-stable-diffusion/issues/966

**Reply by NaotoEnjoyer** [reply]
so issue appears when i make batches, when i do single image gen all works, and tyvm for help, im so glad ppl made colabs for not so high end specs ppl like me

**Reply by Excalibro1** [reply]
Strange issue honestly, and all good I am also glad TheLastBen is doing gods work 🙏
